dotcom-rendering/src/components/Branding.importable.tsx

```diff
@@ -148,6 +151,8 @@ export const Branding = ({ branding, format }: Props) => {
 					aria-label={`Visit the ${branding.sponsorName} website`}
 					onClick={() => trackSponsorLogoLinkClick(sponsorId)}
 					data-testid="branding-logo"
+					data-component={ophanComponentName}
+					data-link-name={ophanComponentLink}
```

Comment:
Most of this PR is testing and shuffling things around, but this is the crucial change that enables Ophan tracking.